src/modules/job-exec/job-exec.c

```diff
@@ -843,13 +843,6 @@ static flux_future_t *jobinfo_start_init (struct jobinfo *job)
     return NULL;
```

Comment:
typo in commit message "notiation" -> "notation"

Reply:
Thanks! fixed the typo an force-pushed.